compare_vcf.py

```diff
@@ -356,24 +363,38 @@ def match_false(augmented_file, files_to_pair_with, out_dir, sample, log_to_file
                                          opts = vcfeval_options, java = java)
 
                         equivalent_variant = utils.get_equivalent_variant(line_split, vcfeval_comparator.get_tp())
-
                         #clean up
                         if os.path.exists(vcfeval_prefix):
                             LOGGER.warn('{0} exists, removing ...'.format(vcfeval_prefix))
                             shutil.rmtree(vcfeval_prefix)
 
                     if i == 0:
+                        AO_RO_DP = {"AO": None, "RO": None, "DP": None}
                         if equivalent_variant:
-                            try:
-                                AO = int(equivalent_variant[-1].split(':')[4].split(',')[0])
-                                RO = int(equivalent_variant[-1].split(':')[2].split(',')[0])
-                            except:
-                                info = "N/A;"
-                            else:
-                                info = str(float(AO)/(AO+RO)) + ';'
+                            for entry in AO_RO_DP:
```

Comment:
Hi @shannog2 could you wrap this part into a (preferably generic) function?

Reply:
Done